app/main.py

```diff
@@ -1,6 +1,36 @@
+import os
+import requests
+
+API_KEY = os.environ.get("API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    if not API_KEY:
+        print("API_KEY environment variable is not set.")
+        return
+
+    params = {
+        "key": API_KEY,
+        "q": CITY
+    }
+
+    try:
+        response = requests.get(BASE_URL, params=params)
+        response.raise_for_status()
+        data = response.json()
+
+        location = data["location"]
+        current = data["current"]
```

Comment:
Consider adding error handling for cases where the API response does not contain the expected 'location' or 'current' keys. This will prevent the script from crashing with a `KeyError` if the API response format changes or if there's an error in the response.